Resources/Prototypes/_CM14/Roles/Jobs/Civilians/survivor.yml

```
@@ -20,17 +20,18 @@
   joinNotifyCrew: false
   supervisors: job-supervisors-nobody
   canBeAntag: false
+  accessGroups:
+  - Colony #for playtest 1-2 only
```

I forgot to make civilian colonist access
```suggestion
  - Colonist
```